README.md

```diff
@@ -27,6 +27,10 @@ Cockroach DB requires some base configuration that can be overridden. (An exampl
 cockroach.host=cockroachdb-proxy
 cockroach.port=26257
 ```
+
+You can configure `--cache` and `--max-sql-memory` cockroachdb flags via
```

Comment:
Looks like this `configuration` section is rather about CA configuration.
Maybe it would be worth to split this section for readability?

Reply:
Updated